os/arch/arm/src/armv7-m/up_hardfault.c

```diff
@@ -84,6 +85,11 @@
 
 #define INSN_SVC0        0xdf00	/* insn: svc 0 */
 
+#ifdef CONFIG_BUILD_FLAT
+extern uint32_t _ram_kernel_text_start;
+extern uint32_t _ram_kernel_text_end;
+#endif
```

Comment:
Why do we need to use these variables directly for flat build?
Can we not use the same api?